datadog/resource_datadog_downtime.go

```diff
@@ -135,7 +136,10 @@ func resourceDatadogDowntime() *schema.Resource {
 				Description:   "When specified, this downtime will only apply to this monitor",
 			},
 			"monitor_tags": {
-				Type:        schema.TypeList,
```

Comment:
Could we make this work without changing the type here. I fear it won't be backward compatible otherwise. What do you think?

Reply:
I only have enough experience with Golan to go by what `datadog_monitor` resource does - https://github.com/terraform-providers/terraform-provider-datadog/commit/6552bcee02a611b4b36e8f7dcdbc1622838bb921#diff-f313670eee4640665f3fee01de4c6ce7R166

It states that set is used because of the ordering problem. After I added all the sorting into this resource it still wasn't working correctly and tests were failing (it was still being reordered by TF). Making it a set fixed that. I assume there is some internal logic terraform that interferes.